client/disputes/utils.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import type { CachedDispute } from 'wcpay/types/disputes';
+import { formatCurrency } from 'wcpay/utils/currency';
+
+interface GetDisputesNoticeStringArgs {
+	/**
+	 * Active disputes (awaiting a response) to generate the notice string for.
+	 */
+	activeDisputes: CachedDispute[];
+}
+
+export const getDisputesNoticeString = ( {
+	activeDisputes,
+}: GetDisputesNoticeStringArgs ): string | null => {
+	const disputeCount = activeDisputes.length;
+
+	if ( disputeCount === 0 ) {
+		return null;
+	}
+
+	if ( disputeCount === 1 ) {
+		const dispute = activeDisputes[ 0 ];
+		const now = moment();
+		const dueBy = moment( dispute.due_by );
+
+		const dueWithin24h =
+			dueBy.diff( now, 'hours' ) > 0 && dueBy.diff( now, 'hours' ) <= 24;
```

Comment:
This logic is going to be useful for the dispute urgency/colour scheme. So maybe it is better to implement this logic all inline in the component now, and factor out small functions for shared needs across multiple components.

Reply:
I've now moved this logic within the `client/overview/task-list/dispute-tasks.ts` file, consistent with logic for other task list items.